rules/gsutil.bzl

```diff
@@ -0,0 +1,71 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Skylark rule for pulling a file from GCS bucket.
+"""
+def _impl(repository_ctx):
+  """Core implementation of gsutil_cp."""
+  target_file = repository_ctx.attr.target_file or repository_ctx.attr.file
+  # Add a top-level BUILD file to export all the downloaded files.
+  repository_ctx.file("BUILD", """
+package(default_visibility = ["//visibility:public"])
```

Comment:
create a BUILD file just like here: https://github.com/bazelbuild/bazel/blob/master/tools/build_defs/repo/http.bzl#L58 (and see my PR for how to set the file path)

Reply:
Thanks for giving me the reference.